hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/validation/EntityType.java

```diff
@@ -21,11 +21,16 @@
  */
 public enum EntityType {
     ACCOUNT,
-    CONTRACT,
-    FILE,
-    NFT,
-    SCHEDULE,
+    ALIAS,
     TOKEN,
     TOKEN_ASSOCIATION,
-    TOPIC
+    NFT,
+    AIRDROP,
+    TOPIC,
+    NODE,
+    FILE,
+    SCHEDULE,
+    CONTRACT_BYTECODE,
+    CONTRACT_STORAGE,
+    STAKING_INFO
```

Comment:
I don't know where this enum is used. Are enum ordinals persisted anywhere? A change like this will affect the ordinals

Reply:
Thanks Artem. Agreed. 
But, this enum is currently used only to recognize if the entityType is `Contract` [here](https://github.com/hashgraph/hedera-services/blob/main/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/ExpiryValidatorImpl.java#L183) and [here](https://github.com/hashgraph/hedera-services/blob/main/hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/TokenHandlerHelper.java#L158), so should not have  any effect.